specs/core/0_beacon-chain.md

```diff
@@ -701,14 +703,23 @@ The per-slot transitions generally focus on verifying aggregate signatures and s
 
 Note: The definitions below are for specification purposes and are not necessarily optimal implementations.
 
+#### `is_active_validator`
+ ```python
+def is_active_validator(validator: ValidatorRecord) -> bool:
```

Comment:
:)

and to think, we almost got rid of `get_active_validator_indices` long ago!